bootstrap-samples/src/main/java/de/agilecoders/wicket/samples/pages/TempusDominusPickerPage.java

```diff
@@ -0,0 +1,350 @@
+                "new TempusDominusConfig()\n" + //
+                "    .withClass(LocalTime.class);",
+                timeConfig
+            );
+        }
+        { // scope
+            TempusDominusConfig iconConfig = new TempusDominusConfig()
+                .withClass(LocalDateTime.class)
+                .withRestrictions(cfg -> cfg
+                        .withDayOfWeekDisabled(0)
+                        .withDayOfWeekDisabled(6)
+                );
+                createBlock(
+                new AbstractTempusDominusWithIcon<>("icon", Model.of((LocalDateTime)null), iconConfig) {
+                    private static final long serialVersionUID = 1L;
+
+                    @Override
+                    protected FormComponent<LocalDateTime> newInput(String wicketId, String dateFormat) {
+                        return new LocalDateTimeTextField(wicketId, dateFormat);
+                    }
+                },
+                "new TempusDominusConfig()\n" + //
+                "    .withClass(LocalDateTime.class)\n" + //
+                "    .withRestrictions(cfg -> cfg\n" + //
+                "            .withDayOfWeekDisabled(0)\n" + //
+                "            .withDayOfWeekDisabled(6)\n" + //
+                "    );"
+            );
+        }
+        form.add(new AjaxButton("submit") {
+            protected void onSubmit(AjaxRequestTarget target) {
+                String tag = languages.getModelObject();
+                final String country = countries.getModelObject();
+                if (!Strings.isEmpty(country)) {
+                    tag += "-" + country;
+                }
+                Locale loc = Locale.forLanguageTag(tag);
+                form.info("'" + loc + "' has been set");
+                Session.get().setLocale(loc);
+                target.add(form);
+            }
+
+            protected void onError(AjaxRequestTarget target) {
+                target.add(feedback);
+            }
+        });
+    }
+
+    private void addStatus(String id, FormComponent<?> input) {
+        final IConverter converter = new IConverter() {
+            public String convertToString(Object value, Locale locale) {
+                return TempusDominusConfig.formatDateISO(value);
+            }
+
+            @Override
+            public Object convertToObject(String value, Locale locale) throws ConversionException {
+                return null;
+            }
+        };
+
+        Label status = new Label(id + "-status", input.getModel()) {
+            @Override
+            public <C> IConverter<C> getConverter(Class<C> type) {
+                return converter;
+            }
+        };
+        input.add(new OnChangeAjaxBehavior() {
+            @Override
+            protected void updateAjaxAttributes(AjaxRequestAttributes attributes) {
+                super.updateAjaxAttributes(attributes);
+                attributes.setSerializeRecursively(true);
```

Comment:
@martin-g maybe there is a chance to override some method on Component so such setting can be applied automatically? :)

Reply:
I don't understand what you suggest here.

Reply:
`AbstractTempusDominusWithIcon` is `FormComponent` consist of panel with single input in it

If I wand to attach `OnChangeAjaxBehavior` to it, it will NOT work :(
`change`  event is being triggered by `input` bubbles to `div`, but there will be no value :(

This can be bypassed using `attributes.setSerializeRecursively(true);` BUT
All users of this component MUST manually add this to their behaviors ... (or attach behaviors to input itself)

What I propose is to add method like: `wantAjaxAttributes` to `FormComponent` so if behavior attached it can attributes will be set automatically :)

I can start discussion at dev@ :))